internal/database/security_event_logs.go

```diff
@@ -54,6 +56,13 @@ type SecurityEvent struct {
 	Timestamp       time.Time
 }
 
+func (e *SecurityEvent) parseArgument() string {
+	if e.Argument == nil {
+		return fmt.Sprintf("%s", []byte(`{}`))
```

Comment:
```suggestion
		return "{}"
```

Or am I missing something?

Reply:
It was a [copy/paste](https://sourcegraph.com/github.com/sourcegraph/sourcegraph/-/blob/internal/database/security_event_logs.go?L94) but why the heck not, that looks way simpler.